<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BooksSelector</class>
 <widget class="QWidget" name="BooksSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="lstSource">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnOk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/ok.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lstDest">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionBookSelect">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать книги</string>
   </property>
   <property name="toolTip">
    <string>Выбрать книги</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../bin/images.qrc"/>
 </resources>
 <connections/>
</ui>
